### PR TITLE
PAN-OS Added !url command and the category indicatorfield

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19974

## Description
added the url command which is based on the PAN-OS URLFiltering service, along with the `category` indicatorfield and mapping from the indicatortype.

## Screenshots
All in the referenced issue.

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests - not yet

### CODE REVIEW

**Comment by bakatzir** [comment]
I'll set a short Demo.
@michalgold / @Arsenikr Would you like to be in it?

**Comment by Arsenikr** [comment]
Yes
Add @EliorKedar as well

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -2223,6 +2223,18 @@ def populate_url_filter_category_from_context(category):
             return context_urls
 
 
+def calculate_dbot_score(url, category):
+    dbot_score = 1
```
Comment (Itay4) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:2227]: why is `1` the default and not `0` ?

**Comment by Itay4** [comment]
is this still in progress if there are no tests?

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 1 above)
Comment (bakatzir) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:2227]: Not really, i'll need security help.
I'm sure the categoris list I have here is partial.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -2242,6 +2255,20 @@ def panorama_get_url_category_command(url_cmd: str):
         context_urls = populate_url_filter_category_from_context(category)
         categories_dict[category] = list((set(categories_dict[category])).union(set(context_urls)))
 
+        score = calculate_dbot_score(url,category)
+        dbot_score = Common.DBotScore(
+            indicator=url,
+            indicator_type=DBotScoreType.URL,
+            integration_name='PAN-OS',
```
Comment (Itay4) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:2262]: i think this needs to match the integration brand/id - `Panorama`

**Comment by bakatzir** [comment]
@Itay4 Will add test.
I want to see if we need mapping to more indicatortypes, or generate more indicatorfield beforehand.

File: Packs/Base/ReleaseNotes/1_1_4.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+- __CommonServerPython__
+Added the category field to the URL indicator
```
Comment (amshamah419) [inline Packs/Base/ReleaseNotes/1_1_4.md:4]:
```suggestion
Added the *category* field to the URL indicator.
```

File: Packs/CommonTypes/ReleaseNotes/1_5_0.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+#### IndicatorFields
+##### Category
+Added the category indicator field.
+
+#### Indicator Types
+##### URL
+Added a mapping to the category indicator field.
```
Comment (amshamah419) [inline Packs/CommonTypes/ReleaseNotes/1_5_0.md:8]:
```suggestion
Added mapping for the category indicator field.
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 2 above)
Comment (Arsenikr) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:2273]: it is always PAN-OS throughout our integrations/playbooks etc so i would keep it for consistency

**Comment by bakatzir** [comment]
No actual diff with AccessInvestigation, merging.

**Pull request merged**